src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+################
+#    Chests    #
+################
+
+tile.ironchest.chest.iron.name=Rautainen arkku
+tile.ironchest.chest.gold.name=Kultainen arkku
+tile.ironchest.chest.diamond.name=Timanttinen arkku
+tile.ironchest.chest.copper.name=Kupari arkku
+tile.ironchest.chest.silver.name=Hopea arkku
+tile.ironchest.chest.crystal.name=Kristallinen arkku
+tile.ironchest.chest.obsidian.name=Obsidiaani arkku
+tile.ironchest.chest.dirtchest9000.name=Multa arkku 9000!
+
+item.ironchest.chest.iron_gold.name=Parannus: raudasta kultaisen
+item.ironchest.chest.gold_diamond.name=Parannus: kultaisesta timanttisen
+item.ironchest.chest.copper_silver.name=Parannus: kuparisesta hopean
+item.ironchest.chest.silver_gold.name=Parannus: hopeasta kultaisen
+item.ironchest.chest.copper_iron.name=Parannus: kuparisesta rautaisen
+item.ironchest.chest.diamond_crystal.name=Parannus: timanttisesta krystallisen
```

Comment:
Kristalli or krystalli.

Reply:
https://fi.wikipedia.org/wiki/Kristalli

Yes. Here I do wrong translate. Correct is Kristalli